### PR TITLE
Bytearray declaration change.

### PR DESCRIPTION
**Pull request by jacqueswww**
### - What I did
Implement VIP #714 .

### - How I did it
See parse_type

### - How to verify it
Tests verify it basically.

### - Description for the changelog
Changed bytes declaration from 'bytes <= n' to 'bytes[n]'.

### CODE REVIEW

**Comment by fubuloubu** [comment]
Coverage change? weird.

**Comment by jacqueswww** [comment]
Strange, will hunt it down! :)

File: vyper/functions/functions.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def _convert(expr, context):
     #     assert typ in ('int128', 'uint256', 'address', 'bytes')
```
Comment (fubuloubu) [inline vyper/functions/functions.py]: Why is this stuff commented out?

Comment (jacqueswww) [reply vyper/functions/functions.py]: Remnants of the `convert` PR a while a go. Will remove it in a separate PR.

Comment (jacqueswww) [reply vyper/functions/functions.py]: Created https://github.com/ethereum/vyper/pull/744.

**Pull request merged**